sopel_remind/plugin.py

```diff
@@ -220,3 +220,42 @@ def remind_at(bot: SopelWrapper, trigger: Trigger):
         time=when,
     )
     bot.reply('I will remind you that at %s' % display_when)
+
+
+@plugin.command('reminders clear')
+def reminders_clear(sopel: SopelWrapper, trigger: Trigger) -> None:
+    """Clear all your channel or private reminders.
+
+    When used in a channel, this will remove all the reminders you have defined
+    in that channel. When used in a private message, this will remove your
+    private reminders instead.
+
+    In both case you'll get a private NOTICE with the number of reminders
+    removed (if any).
+    """
+    nick = trigger.nick
+    destination = trigger.sender
+
+    if option := trigger.group(2):
```

Comment:
Nicks and channels can't contain whitespace, right?


```suggestion
    if option := trigger.group(3):
```

Either this, or `.strip()` group 2.